TaskWithNullptr/TaskWithNullptr.cpp

```diff
@@ -0,0 +1,16 @@
+﻿#include <iostream>
+
+int main()
+{
+	try
+	{
+		double* ptr = nullptr; // create a null pointer 
+		std::cout << *ptr; // dereference a null pointer, exception will trigger
+	}
+	catch (const std::exception&) // catch does not work because dereferncing null pointer is undefined behaviour
```

Comment:
Read about multiple catch blocks and add one more catch (...) {} down the code to catch the dereferencing. This way you learn we can differentiate exceptions and handle them differently.